Exercises/4-difference.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-// Find difference of two arrays
-// elements from array1 that are not includes in array2
-
-const difference = (array1, array2) => [];
+const difference = (array1, array2) => array1.filter(elem => {
```

Comment:
Try to reduce complexity:
- Don't create intermediate array `duplicate`
- You can implement it using just resulting array and for..of loop

Reply:
done